signed-playback-lambda/README.md

```diff
@@ -0,0 +1,46 @@
+# Lambda for Signed Playback
+
+Mux Playback IDs have two types: `public` and `signed`. 
+`public` playback URLs can be watched anywhere, any time. 
+`signed` playback URLs require a Mux [signing key](https://docs.mux.com/reference#url-signing-keys)
+which is used to generate a token via [JSON Web Tokens](https://jwt.io/).
+
+This example includes an [AWS Lambda](https://aws.amazon.com/lambda/) function that can receive a Mux [playback ID](https://docs.mux.com/reference#playback-ids)
+and return a signed url that's ready for use.
+
+## In Use
+```bash
+# Generate signed playback URL
+curl -X POST \
+    -H "Content-Type: application/json" \
+    -d '{"playbackId": "1234"}' \
+    https://example.com/my-example-lambda
+
+# Response
+{
+    "playbackId": "1234",
+    "token": "some-token",
+    "signedUrl": "https://stream.mux.com/1234.m3u8?token=some-token"
+}
+```
+
+Generate a signed playback URL by making a POST request to your lambda. 
+Pass `playbackId` and other configuration in the body of the request.
+
+## Deployment
+1. Run `yarn build` to bundle all js code and create a zip file that can be uploaded to AWS Lambda.
+2. Create a new empty function in AWS Lambda.
```

Comment:
It might be worth adding a `Make sure to Node.js is selected for the runtime option` or something along those lines (that _should_ be obvious, but you never know)